src/components/NavBar/PostButtonModal.tsx

```diff
@@ -7,7 +7,10 @@ const styles = {
     ".MuiDialog-scrollPaper": { alignItems: "flex-start" },
   },
   dialogContent: {
-    padding: 0,
+    padding: 1,
```

Comment:
Do we need this stuff?

Reply:
nah this is out of scope and random lol ima revert it